src/Modal/modalManager.js

```diff
@@ -45,13 +35,22 @@ browser has a chance to reach the <body>.
 Please move the import at the end of the <body>.
```

Comment:
'to the end'

Reply:
oh sry over missed that as i didn't change it :D